README.md

```diff
@@ -30,6 +30,13 @@ All source files are UTF-8 encoded with Unix line-endings (``LF``, ``\n``). They
 
 We maintain a style guide for the Currencycloud API documentation. See [STYLE_GUIDE.md](STYLE_GUIDE.md).
 
+## Deployed API
+The currently deployed API will be the latest tagged release version of currencycloud-swagger. 
```

Comment:
I would reword as follows: 

"The current version of the API is the latest tagged release" - and probably add a hyperlink to releases for convenience...